src/framework/audio/internal/audiobuffer.cpp

```diff
@@ -116,10 +116,11 @@ void AudioBuffer::fillup()
         return;
     }
 
+    static float buffer[FILL_SAMPLES * 2] = {};
```

Comment:
2 needs replaced to `streamCount` (or const AUDIO_CHANNELS)

Reply:
oh, wow
I've started to use AUDIO_CHANNELS because of this line and completely forgot to update it :)